christin/index.js

```diff
@@ -0,0 +1,21 @@
+const colors = [
+  "red",
+  9999,
+  "green",
+  "blue",
+  777,
+  "yellow",
+  "black",
+  "purple",
+  "brown",
+];
+
+colors[1] = [""];
```

Comment:
The way you solved is correct, But I am going to show you the best solution, so imagine you have hundred or maybe thousands of elements in an Array so how you can switch all numbers to string! Here is the solution!
const colors = ["red", 9999, "green", "blue",777, "yellow", "black", "purple", "brown"];
let colorPicker = Math.floor(Math.random() * colors.length);
let filtered = typeof colors[colorPicker] === "number" ? "" : colors[colorPicker];
console.log(filtered);

Reply:
ok thanks , how can i teach myself to think more like that?@ciamac-da  just to get it right :
here you asked - if the datatype is a number do "" if not git back the color, right? 
  let filtered = typeof colors[colorPicker] === "number" ? "" : colors[colorPicker];

Reply:
Hey Christin, no prob, actually its about time and you have to stay keep on the ball and try to see more algorithms, try to watch tutorials as much as you can and try to write the same code parallel as you watching tutorials.
So back to the code and let filtered, so here the logic is if typeof randomed colors array which is stored in colors[colorPicker] was number then switch it to ""(empty string) , otherwise give me colors[colorPicker] as result, which contains strings becuase we already replaced numbers with string using if statement